record/record_test.go

```diff
@@ -320,10 +321,10 @@ var _ = Describe("Out New Relic", func() {
 			// Then
 			Expect(err).To(BeNil())
 			Expect(packagedRecords).To(Not(BeNil()))
-			// The 20 records are compressed into 8 byte buffers, as overall they exceed 1MB. Note that this is
+			// The 20 records are compressed into 12 byte buffers, as overall they exceed 1MB. Note that this is
 			// a deterministic result (unless the gzip implementation changes), since we're always using the same
 			// seed when generating the random messages, which will lead to the same messages always being sent.
-			Expect(packagedRecords).To(HaveLen(8))
+			Expect(packagedRecords).To(HaveLen(12))
```

Comment:
I need to update this test as the compression library has changed on recent go versions.